.vscode/launch.json

```diff
@@ -70,20 +71,37 @@
                 "order": 1
             },
             "console": "internalConsole",
-            "cwd": "${workspaceRoot}",
+            "cwd": "${workspaceFolder}",
             "runtimeExecutable": null,
             "env": {
                 "NODE_ENV": "development"
             },
         }
     ],
     "compounds": [
+        {
```

Comment:
We already have one configuration named "Editor With Devtools".
The compound "Editor" is now deprecated thanks to your contribution, I think we can replace it and keep only "Editor" as compound :)

Reply:
Done!